src/components/modal/modal.tsx

```diff
@@ -134,6 +130,7 @@ export class Modal
     setUpLoadableComponent(this);
     // when modal initially renders, if active was set we need to open as watcher doesn't fire
     if (this.open) {
```

Comment:
there is the prop "open" and the state "isOpen" how are these different and can we choose just one?

Reply:
`isOpen` was used in conjunction with the `active `prop, which has been deprecated in favor of `open`. It’s still used in other components as well. It would be out of scope for this issue, but I can look at it separately in another one to determine whether we need to keep it or refactor out.